core/settings/base.py

```diff
@@ -144,5 +145,10 @@
 # Name of the URL where a user may login
 LOGIN_URL = 'auth_login'
 
-
 AUTH_USER_MODEL = 'register.User'
+
+OAUTH2_PROVIDER = {
+    'SCOPES': {'email': 'Viewing your email address.'},
+    'READ_SCOPE': ['email'],
```

Comment:
It would be possible for sure, but I'm really not into making something more generic – aka more complex and less explicit – while we don't have the need.